ionoscloud/resource_logging_pipeline.go

```diff
@@ -163,6 +175,24 @@ func pipelineUpdate(ctx context.Context, d *schema.ResourceData, meta interface{
 }
 
 func pipelineImport(ctx context.Context, d *schema.ResourceData, meta interface{}) ([]*schema.ResourceData, error) {
-	pipelineRead(ctx, d, meta)
+	parts := strings.Split(d.Id(), ":")
+	if len(parts) != 2 {
+		return nil, fmt.Errorf("expected ID in the format location:id")
+	}
+
+	location := parts[0]
+	if !slices.Contains(logging.AvailableLocations, location) {
+		return nil, fmt.Errorf("invalid location: %v, location must be one of: %v", location, logging.AvailableLocations)
+	}
+
+	if err := d.Set("location", parts[0]); err != nil {
+		return nil, fmt.Errorf("failed to set location Kafka Cluster for import: %w", err)
```

Comment:
It should be `Logging Pipeline`, not `Kafka Cluster` :))))
```suggestion
		return nil, fmt.Errorf("failed to set location Logging Pipeline for import: %w", err)
```